jetty-ee9/jetty-ee9-maven-plugin/src/main/java/org/eclipse/jetty/ee9/maven/plugin/ServerSupport.java

```diff
@@ -68,11 +66,7 @@ public static void configureHandlers(Server server, List<ContextHandler> context
         if (contexts == null)
         {
             contexts = new ContextHandlerCollection();
-            Handler.Collection handlers = server.getDescendant(Handler.Collection.class);
-            if (handlers == null)
-                server.setHandler(new Handler.Collection(contexts, new DefaultHandler()));
-            else
-                handlers.addHandler(contexts);
```

Comment:
Causes #12207, also decreasing the accuracy of the Javadoc a few lines above, which (now incorrectly) states:

> Also put in a `DefaultHandler` so we get a nicer page than a 404 if we hit the root and the webapp's context isn't at root.